Docker/create_tarfiles.py

```diff
@@ -24,104 +27,163 @@
     "ko.tsv": ".ko.txt",
     "gene_phylogeny.tsv": ".phylodist.txt",
     "ec.tsv": ".ec.txt",
+    "supfam.gff": ".supfam.txt",
     "pfam.gff": ".pfam.txt",
     "tigrfam.gff": ".tigr.txt",
     "cath_funfam.gff": ".cathfunfam.txt",
     "smart.gff": ".smart.txt",
-    "supfam.gff": ".supfam.txt",
     "functional_annotation.gff": ".gff"
 }
 
-# Filter functions
-def filter_gff(input_file, output_file, contig_ids):
-    with open(output_file, "w") as out_file:
-        with open(input_file) as in_file:
-            for line in in_file:
-                contig_id = line.rstrip().split()[0]
-                if contig_id in contig_ids:
-                    out_file.write(line)
 
-def filter_faa(input_file, output_file, contig_ids):
-    with open(output_file, "w") as out_file:
-        with open(input_file) as in_file:
-            for line in in_file:
-                if line.startswith(">"):
-                    file_id = line[1:].rstrip().split()[0]
-                    contig_id = "_".join(file_id.split("_")[0:-2])
-                    contig_prefix = "-".join(file_id.split("-")[0:2])
-                if contig_prefix.startswith("nmdc:") and len(contig_ids) > 0 and contig_prefix not in contig_ids[0]:
-                    print(f"{contig_prefix} not part of {contig_ids[0]}, Please check the mapping file.", file=sys.stderr)
-                    sys.exit(1)
-                if contig_id in contig_ids:
-                    out_file.write(line)
-
-def filter_inp(input_file, output_file, contig_ids):
-    with open(output_file, "w") as out_file:
-        with open(input_file) as in_file:
-            for line in in_file:
-                file_id = line.rstrip().split()[0]
-                contig_id = "_".join(file_id.split("_")[0:-2])
-                if contig_id in contig_ids:
-                    out_file.write(line)
+def get_contig_faa(line, contig_id):
```

Comment:
```suggestion
def get_contig_faa(line, contig_ids):
```
Otherwise there are unresolved references in lines 44-46

Reply:
Weird.    Let me look since this was working in my tests.

Reply:
I dropped the use of contig_ids all together.  It was left over.